python/lsst/sconsUtils/state.py

```diff
@@ -172,14 +172,20 @@ def _initEnvironment():
     if env['PLATFORM'] == 'darwin':
         env['LDMODULESUFFIX'] = ".so"
         if not re.search(r"-install_name", str(env['SHLINKFLAGS'])):
-            env.Append(SHLINKFLAGS=["-Wl,-install_name", "-Wl,${TARGET.file}"])
+            env.Append(SHLINKFLAGS=["-install_name", "@rpath/${TARGET.file}"])
```

Comment:
I've never used `conda-build`, and I'm not sure I understand the point that @mjuric  is raising here -- my beginner-level grasp of how this is supposed to work agrees with @timj.

However: this doesn't seem like it's directly related to the issue at hand, it doesn't solve a problem we're having this week, and if there's a risk it could interfere with the Anaconda build, my suggestion is to drop it.